add-on/panels/css/panel.css

```diff
@@ -96,6 +96,56 @@ body {
   border-radius: 3px;
 }
 
+/* Rename closing room panel */
+.rename-container {
+  margin: 15px;
+  padding-left: 45px;
```

Comment:
You'll need to do rtl versions for this and other places where you use left/right, as currently the display looks messed up if I load the showcase in rtl mode.